pathway_indexer/parser.py

```diff
@@ -97,22 +97,53 @@ def has_changes(row):
     unchanged_html_files = html_df[~html_df["HasChanged"]]
 
     # Copy unchanged HTML files and remove them from input directory
-    for _, row in unchanged_html_files.iterrows():
-        print("Skipping unchanged file:", row["Filepath"])
-        pathname = os.path.basename(row["Filepath"]).replace(".html", ".md")
-        src_path = os.path.join(last_data_json["last_folder_crawl"], "out", "from_html", pathname)
-        dst_path = os.path.join(out_folder, "from_html", pathname)
-
```

Comment:
Could we add if normalized src_path == normalized dest_path then continue here? And avoid all of the other code?